n1netails-api/src/main/java/com/n1netails/n1netails/api/constant/ProjectSecurityConstant.java

```diff
@@ -20,6 +20,7 @@ public class ProjectSecurityConstant {
             "/ninetails/user/register",
             "/ninetails/auth/passkey/**",
             "/ninetails/alert",
+            "/ninetails/notification/**",
```

Comment:
should probably consider removing this so only authorized users can send an email